src/mavedb/routers/permissions.py

```diff
@@ -0,0 +1,65 @@
+import logging
+from enum import Enum
+
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.orm import Session
+from typing import Union, Optional
+
+from mavedb import deps
+from mavedb.lib.authentication import get_current_user, UserData
+from mavedb.lib.permissions import has_permission, Action
+from mavedb.lib.logging import LoggedRoute
+from mavedb.lib.logging.context import logging_context, save_to_logging_context
+from mavedb.models.experiment import Experiment
+from mavedb.models.experiment_set import ExperimentSet
+from mavedb.models.score_set import ScoreSet
+
+router = APIRouter(
+    prefix="/api/v1/permissions",
+    tags=["permissions"],
+    responses={404: {"description": "Not found"}},
+    route_class=LoggedRoute,
+)
+
+logger = logging.getLogger(__name__)
+
+
+class ModelName(str, Enum):
+    experiment = "experiment"
+    experiment_set = "experiment-set"
+    score_set = "score-set"
+
+
+@router.get(
+    "/user-is-permitted/{model_name}/{urn}/{action}",
+    status_code=200,
+    response_model=bool
+)
+async def check_authorization(
+    *,
+    model_name: str,
```

Comment:
Since you went through the trouble of creating an enum for the model name, we should probably use it here instead of this being a string
```suggestion
    model_name: ModelName,
```

Reply:
This one done.